collector-config.yaml

```diff
@@ -8,12 +8,14 @@ receivers:
           allowed_origins:
             - http://*
             - https://*
+            - "*"
```

Comment:
@Pehesi97 Why do we need to allow origins other than `http://*` and `https://*` (which already seem pretty permissive)?

Reply:
Honestly, I added that line for testing only. I was having problems on connecting the frontend and backend to the collector, and this was an attempt to debug.
It should be OK to remove that line.

Reply:
We will clean this up with #186 #187 and #188